modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastRatingsViewModel.kt

```diff
@@ -62,7 +63,13 @@ class PodcastRatingsViewModel
             try {
                 ratingsManager.refreshPodcastRatings(uuid)
             } catch (e: Exception) {
-                Timber.e(e, "Failed to refresh podcast ratings")
+                val message = "Failed to refresh podcast ratings"
+                // don't report missing rating or network errors to Sentry
+                if (e is HttpException || e is IOException) {
+                    Timber.i(e, message)
+                } else {
+                    Timber.e(e, message)
+                }
```

Comment:
This should stop the following Sentry error as I can see the previous call is the ratings endpoint.
```
HttpException
HTTP 404 
```